RadarSDK/Include/Radar.h

```diff
@@ -765,6 +774,34 @@ typedef void (^_Nonnull RadarRouteMatrixCompletionHandler)(RadarStatus status, R
                        units:(RadarRouteUnits)units
            completionHandler:(RadarRouteMatrixCompletionHandler)completionHandler NS_SWIFT_NAME(getMatrix(origins:destinations:mode:units:completionHandler:));
 
+/**
+ Sends a custom event.
+
+ @param name The name of the event.
+ @param metadata The metadata associated with the event.
+ @param completionHandler A completion handler.
+
+ @see https://radar.com/documentation/api#create-a-custom-event
```

Comment:
Create/send/log? Should we use send everywhere? Will it be clear to developers that you "generate" events like geofence entries but "send" custom events?

Reply:
good call - will amend here and in docs to make the difference explicit